weld/webservices/pom.xml

```diff
@@ -4,28 +4,63 @@
    <modelVersion>4.0.0</modelVersion>
 
    <parent>
+      <!--move back to weld-parent when other weld components moves to jakarta namespace-->
       <groupId>org.wildfly</groupId>
-      <artifactId>wildfly-weld-parent</artifactId>
+      <artifactId>wildfly-parent</artifactId>
```

Comment:
Is this because Weld has not been migrated yet?

Reply:
I think this is because @jimma is wanting to avoid both the parent dependencyManagement (legacy-ee) and this modules (standard-ee) both taking effect.

I've never tried but I'm pretty sure you can't use combine-children=override to prevent this, as that is for children of plugin 'configuration' elements.

Reply:
@bstansberry @jamezp  Yes. That's reason. The other weld modules are still import the legacy-ee dependencies, so I have to change the parent name . After the other weld modules are migrated to jakarta namespace, we can change back to `wildfly-weld-parent`.